cli/hse_cli.c

```diff
@@ -170,7 +170,8 @@ static cli_cmd_func_t cli_hse_storage_profile;
 struct cli_cmd        cli_hse_storage_commands[] = {
     { "add", "Add a new media class storage to an existing offline KVDB", cli_hse_storage_add, 0 },
     { "info", "Display storage information about a KVDB", cli_hse_storage_info, 0 },
-    { "profile", "Profile storage path to determine throttle policy", cli_hse_storage_profile, 0 },
+    { "profile", "Profile capacity storage path to determine throttle policy",
```

Comment:
And maybe "Profile storage to determine..."  instead of "Profile storage path to determine ..."

Reply:
Or "Profile capacity storage to determine..." since the intent (as documented) is to profile the storage hosting the capacity media class when it exists (which is all KVDB configs except pmem-only).